### DISCUSSION TITLE
Add a new element type

### DISCUSSION THREAD

**Discussion by tctheodosiou** [idaholab/moose#26378] [Q&A Modules: Solid mechanics] [answered]
Hi all,
When generating a mesh with GeneretedMeshGenerator, I can choose one of the element type available (EDGE, QUAD etc.), as defined by libmesh. Is it possible to define my own element types? I mean something like UEL in Abaqus.
The use of GeneretedMeshGenerator is not mandatory; I could use a different Mesh Generator, an external mesh, whatever is needed.

**Comment by GiudGiud** [comment, marked as answer]
Hello

No you cannot unless you implemented it in the GeneratedMeshGenerator.
But if you do implement it I would recommend you create a new mesh generator for that

Guillaume

**Comment by tctheodosiou** [comment]
Thank you.
